odl/test/tomo/geometry/geometry_test.py

```diff
@@ -665,38 +593,111 @@ def test_helical_cone_flat_props(shift):
                              [0, 0, 1]])
 
     # pitch_offset
-    geom = odl.tomo.HelicalConeFlatGeometry(apart, dpart, src_rad, det_rad,
-                                            pitch, pitch_offset=0.5)
+    geom = odl.tomo.ConeFlatGeometry(apart, dpart, src_rad, det_rad,
+                                     pitch=pitch, pitch_offset=0.5)
```

Comment:
I agree that `pitch_offset` sounds like the pitch is offset. I'll probably go for a longer name like `offset_along_axis` since it's an advanced option anyway.